mino/minows/address_test.go

```diff
@@ -9,65 +9,78 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-const (
-	AddrAllInterface = "/ip4/0.0.0.0/tcp/80"
-	AddrLocalhost    = "/ip4/127.0.0.1/tcp/80"
-	AddrHostname     = "/dns4/example.com/tcp/80"
-)
-const (
-	PID1 = "QmaD31nEzFGwD8dK96UFWHtTYTqYJgHLMYSFz4W4Hm2WCU"
-	PID2 = "QmVt9t5Tk2uEoA4CDbKNCVxqrut8UXmWHXvFZ8wFZ3ghhv"
-)
-
-func TestNewAddress(t *testing.T) {
+func Test_newAddress(t *testing.T) {
+	const pid1 = "QmaD31nEzFGwD8dK96UFWHtTYTqYJgHLMYSFz4W4Hm2WCU"
+	const pid2 = "QmVt9t5Tk2uEoA4CDbKNCVxqrut8UXmWHXvFZ8wFZ3ghhv"
+	const addrAllInterface = "/ip4/0.0.0.0/tcp/80"
+	const addrLocalhost = "/ip4/127.0.0.1/tcp/80"
+	const addrHostname = "/dns4/example.com/tcp/80"
```

Comment:
Please add some protocol identifier like `p2pws://ip4/0.0.0.0/tcp/80`

Reply:
https://docs.libp2p.io/concepts/fundamentals/addressing/

Reply:
I'm more concerned with mino-identifiers. Like the `grpc://` and `grpc://` ones used for minogrpc.

On second thought I'm not sure it makes that much sense, as one dela-cli can only handle one mino-implementation anyway. Let's leave it like that.